datasets.py

```diff
@@ -694,3 +694,146 @@ def from_path(self, file_path):
             anns = [None, [0], [country], file_path, None]
 
         return image, anns
+
+#I made a dataset for CelebA with notes about details I wanted clarification on or about potential issues
```

Comment:
this comment can be removed, and clarifications moved into the general dataset class above